### DISCUSSION TITLE
Eversense support

### DISCUSSION THREAD

**Discussion by k0rtalian** [NightscoutFoundation/xDrip#2203] [General]
I have look everywhere and I did not see any information about support with using Eversense. I found some articles from 2019, but nothing current. Does anyone have eversense working on xdrip and if so can I get where that information is.

**Comment by Der-Schubi** [comment]
AFAIK you need ESEL: https://github.com/BernhardRo/Esel/blob/master/apk/esel.apk
The AAPS docs have some information: https://androidaps.readthedocs.io/en/latest/Hardware/Eversense.html
Just ignore the AAPS part ;)
